exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/VarLenFixedEntryReader.java

```diff
@@ -43,7 +43,7 @@ final VarLenColumnBulkEntry getEntry(int valuesToRead) {
 
     final int expectedDataLen = columnPrecInfo.precision;
     final int entrySz = 4 + columnPrecInfo.precision;
-    final int readBatch = Math.min(entry.getMaxEntries(), valuesToRead);
+    final int readBatch = getFixedLengthMaxRecordsToRead(valuesToRead);
```

Comment:
Would it be nicer to pass the *entrySz* as a parameter to *getFixedLengthMaxRecordsToRead()* , instead of recreating it ?
  E.g., in case it ever changes. Or maybe it'll look clearer.